packages/components/psammead-section-label/src/index.stories.jsx

```diff
@@ -48,3 +53,7 @@ storiesOf('Components|SectionLabel', module)
     ),
     { notes, knobs: { escapeHTML: false } },
   );
+
```

Comment:
Could you please also vary the `linkText="See All"` text int he story above, since in the RTL example it looks quite broken seeing the English LTR text here. 
<img width="1440" alt="Screenshot Section Label example" src="https://user-images.githubusercontent.com/3028997/67415304-b45a8a00-f5bc-11e9-8104-656952c7e031.png">
http://localhost:8180/?path=/story/components-sectionlabel-rtl--rtl-with-a-link

Reply:
What would be the best way to do this? Should I select the first 1 or 2 words as the `See All` translation?

Reply:
Yes that should work, since we aren't using proper translations for Psammead stories.